v1/ocagent.go

```diff
@@ -117,13 +114,24 @@ func (ae *Exporter) Start() error {
 	return err
 }
 
+func (ae *Exporter) prepareAgentAddress() string {
```

Comment:
A random thought: do we want to add a fallback here? If the agent address provided by user is not valid (e.g port is already taken), can we fall back to default address?

Reply:
In deed. So inside the body of that function, we use the default agent port and address if things aren't right i.e.
<img width="496" alt="screen shot 2018-09-11 at 11 54 48 am" src="https://user-images.githubusercontent.com/4898263/45378004-846b4580-b5b9-11e8-845d-aa44a437de02.png">